libs/vre/shared/app-resource-page/src/lib/resource-representation.component.ts

```diff
@@ -19,14 +19,12 @@ import { filter, map } from 'rxjs/operators';
     <app-still-image
       #stillImage
       class="dsp-representation stillimage"
-      [attachedProject]="attachedProject$ | async"
       *ngSwitchCase="representationConstants.stillImage"
       [resource]="resource.res" />
     <app-still-image
       #stillImage
       class="dsp-representation stillimage"
       *ngSwitchCase="representationConstants.externalStillImage"
-      [attachedProject]="attachedProject$ | async"
```

Comment:
I think the attached project is required, sometimes it is acquired from a parent resource.

Reply:
It was not needed so I've removed it

Reply:
It is needed, you can see it by clicking download image in this resource i.e.: http://localhost:4200/resource/0803/qEKDZ8HuX2GEmsd-h9WIVQ

there is a fix, you can take it from there: 
https://github.com/dasch-swiss/dsp-das/pull/1853/files

Reply:
your example does not work on my machine, and that variable was marked as unused by my IDE.

Please provide a linear issue to show me the bug, otherwise it's marked as resolved.